README.md

```diff
@@ -15,14 +15,14 @@
 
 ## Summary
 
-- :zap: **Blazing fast performance:** running tasks in parallel using multiple processes.
-- :electric_plug: **Highly modular plugin system:** making it possible to change almost anything.
+- :zap: **Blazingly-fast performance:** running tasks in parallel using multiple processes.
+- :electric_plug: **A Highly modular plugin system:** making it possible to change almost anything.
```

Comment:
I'm not sure we need the `A` addition.

Reply:
My reasoning was this: when you wrote "Blazing fast performance" it was clear the performance was **of** `Haste`, i.e "Haste has blazing fast performance". On the other hand, "Haste has highly modular plugin system" sounded kind of awkward in my mind :)